packages/components/src/components/tabs-bar/tabs-bar.tsx

```diff
@@ -153,7 +156,7 @@ export class TabsBar {
     return (
       <PrefixedTagNames.pScroller
         class="scroller"
-        aria={{ role: 'tablist' }}
+        aria={{ role: this.tabsAreButtons ? 'tablist' : null }}
```

Comment:
This wouldn't set the aria prop at all and therefore not do unneeded loops, validations, etc. in `p-scroller`.
```suggestion
        {...(this.tabsAreButtons && { aria: { role: 'tablist' } })}
```